### PR TITLE
Adding the vertical() accessor to Plot.Bar.

### PR DESCRIPTION
**Pull request by istarnx**

### CODE REVIEW

File: src/components/plots/barPlot.ts [hunk 1]
```diff
@@ -74,6 +74,15 @@ export module Plot {
     }
 
     /**
+     * Returns true if the plot is vertical.
```
Comment (bluong) [inline src/components/plots/barPlot.ts]: We don't need the doc here

File: test/components/plots/barPlotTests.ts [hunk 2]
```diff
@@ -634,6 +639,11 @@ describe("Plots", () => {
         });
 
       });
+
+      it("vertical() returns false", () => {
+        assert.isFalse(barPlot.vertical());
```
Comment (bluong) [inline test/components/plots/barPlotTests.ts]: Let's put a small message here like Horizontal Bar plots are always not vertical

File: test/components/plots/barPlotTests.ts [hunk 3]
```diff
@@ -270,6 +270,11 @@ describe("Plots", () => {
           svg.remove();
         });
       });
+
+      it("vertical() returns true", () => {
+        assert.isTrue(barPlot.vertical());
```
Comment (bluong) [inline test/components/plots/barPlotTests.ts]: Let's put a small message here like Vertical Bar plots are always vertical

File: src/components/plots/barPlot.ts [hunk 4]
```diff
@@ -74,6 +74,15 @@ export module Plot {
...
+     *
+     * @returns {boolean} true if the plot is vertical
```
Comment (bluong) [inline src/components/plots/barPlot.ts]: This returns should all that needs to be required.

**Comment by istarnx** [comment]
Sure--I can make the changes if you'd like. I based that doc style off of Java standards, which typically call for a description AND return value.

**Comment by istarnx** [comment]
Also, note that other boolean methods in Bar have both a description and return value comment.

**Comment by bluong** [comment]
Based on our recent API discussion, if we have doc that's verbose/redundant it'd be best to avoid it.  Less is more.  We can discuss more if you want

**Pull request closed without merging**